src/11_args.py

```diff
@@ -21,13 +29,18 @@
 a = [7, 6, 5, 4]
 
 # What thing do you have to add to make this work?
-print(f2(a))    # Should print 22
+print(f2(sum(a)))    # Should print 22
 
 # Write a function f3 that accepts either one or two arguments. If one argument,
 # it returns that value plus 1. If two arguments, it returns the sum of the
 # arguments. Google "python default arguments" for a hint.
 
 # YOUR CODE HERE
+def f3(*args):
+    if(len(args) == 1):
+       return sum(args) + 1
+    elif(len(args) == 2):
+        return sum(args)
```

Comment:
a default argument would look something like this `def f3(a, b=1):`. Basically if b is not passed as an argument during the function call, than b automatically equals a value of 1.
```
def f3(a, b=1):
    return a + b
```